src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,12 +25,14 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_NATIONALITY + "NATIONALITY "
+            + PREFIX_TUTORIAL_GROUP + "TUTORIAL GROUP "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_NATIONALITY + "Singapore "
+            + PREFIX_TUTORIAL_GROUP + "09 "
```

Comment:
Should tutorial groups prepend with a letter? (i.e. `W09`) There are some groups that share the same number (e.g. `W14` and `T14`)